csc-site-ssapp/code/pages/kit.html

```diff
@@ -131,7 +131,7 @@
             </div>
 
             <div data-if='@actions.canAssignKit'>
-              <button class='btn-default' data-tag='assign-kit' disabled='@kitModel.kit.studyHasEnded'>
```

Comment:
why did you removed this? Disabling kit assignation should be kept if the study is finished